src/main/resources/application.conf

```diff
@@ -1,13 +0,0 @@
-akka {
-  #loglevel = "DEBUG"
-  #stdout-loglevel = "DEBUG"
-
-  actor {
-    debug {
-      # enable DEBUG logging of actor lifecycle changes
-      lifecycle = on
-      unhandled = on
-      autoreceive = on
-    }
-  }
-}
```

Comment:
This file is not needed. It's used in applications, but nakadi-klients is a library, thus providing a reference.conf instead.